yt_dlp/extractor/magentamusik.py

```diff
@@ -0,0 +1,41 @@
+from .common import InfoExtractor
+
+
+class MagentaMusikIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?magentamusik\.de/(?P<id>.*)'
+
+    _TESTS = [{
+        'url': 'https://www.magentamusik.de/marty-friedman-woa-2023-9208205928595409235',
+        'md5': 'd82dd4748f55fc91957094546aaf8584',
+        'info_dict': {
+            'id': '9208205928595409235',
+            'ext': 'mp4',
+            'title': 'Marty Friedman: W:O:A 2023',
+            'duration': 2760
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        video_id = self._html_search_regex(r'"assetId":"[^\d]+([0-9]+)"', webpage, 'video_id')
```

Comment:
Could potentially be made more robust with something like this:
```suggestion
        player_config = self._search_json(r'data-js-element="o-video-player__config">\s*', webpage, 'player config', display_id)
        if 'assetId' not in player_config:
            raise ExtractorError('No assetId found in player config')
        asset_details = self._download_json(
            f'https://wcps.t-online.de/cvss/magentamusic/vodclient/v2/assetdetails/58938/{player_config["assetId"]}',
            display_id, note='Downloading asset details')
        video_id = traverse_obj(asset_details, ('content', 'partnerInformation', ..., 'reference'), get_all=False)
```
`asset_details` also contains some additional metadata.
Curious to hear if others think this would be worth it. Maybe the simple regex approach could be kept as a fallback.